README.md

```diff
@@ -18,6 +18,11 @@ threshold while ignoring allowlisted advisories.
 - _(Optional)_ Yarn ^1.12.3 || Yarn >=2.4.0
 - _(Optional)_ PNPM >=4.3.0
 
+## Known issues
+
+- There is a bug with Yarn Classic that is causing audit to skip `devDependencies`.
+  See [this issue](https://github.com/yarnpkg/yarn/issues/7047) for more information.
```

Comment:
Nice job! Would consider this a "limitation" rather than an "issue"? Also, this must reference that this is only for Yarn workspaces. I also think it might be more relevant to add after the `## Set up` section so that the reader has more context of what we are even talking about. However, that does risk being overlooked.... Thoughts?

Attempt:


## Limitations

- Yarn Classic workspaces does not audit `devDependencies`. See [this issue](https://github.com/yarnpkg/yarn/issues/7047) for more information.

Reply:
Hmm yeah I suppose this would be considered a limitation now since it won't be fixed anymore.  I placed it near the top originally because of visibility, but I'm cool if we place it after the setup.